src/models/event.ts

```diff
@@ -329,4 +336,26 @@ export default class Event extends RestfulModel {
         return response.ics;
       });
   }
+
+  private validate(): void {
+    if (
+      this.conferencing &&
+      this.conferencing.details &&
+      this.conferencing.autocreate
+    ) {
+      throw new Error(
+        "Cannot set both 'details' and 'autocreate' in conferencing object."
+      );
+    }
+    if (
+      this.capacity &&
+      this.capacity != -1 &&
+      this.participants &&
+      this.participants.length <= this.capacity
```

Comment:
I don't think participants are included in the event capacity, since they're part of the meeting by default. cc @tbadun

Reply:
^ that's just a weird internal verbiage thing, the participants list in an event is the sum total of organizers, participants, attendees, and invitees

Reply:
our cc-side validation looks like:
```python
def validate_capacity_request(self, capacity):
    # type: (int) -> None
    if (not isinstance(capacity, int)) or (
        (capacity == 0) or (capacity > 1000) or (capacity < -1)
    ):
        raise InputError(
            "Allowed capacity is between 1-1000 or -1 for unlimited: {}".format(
                capacity
            )
        )
    # only count none-organizer
    num_participants = len(
        [
            participant
            for participant in self.participants
            if participant.get("role") != "organizer"
        ]
    )

    if 0 < capacity < num_participants:
        raise InputError("Participants over capacity: {}".format(capacity))
```

Reply:
Cool, just wanted to double check 👍

Reply:
"participants" means nothing and everything at the same time, just to make communication around scheduler the most fun possible

Reply:
@ozsivanov @tbadun thank you both!